### ISSUE TITLE
pytorch-quantization example is incompatible with latest torchvision version

### ISSUE THREAD

**Issue by SebastianPinzello** [NVIDIA/TensorRT#3495]
## Description

I'm trying to run the following pytorch-quantization example:
[tools/pytorch-quantization/examples/calibrate_quant_resnet50.ipynb](https://github.com/NVIDIA/TensorRT/blob/master/tools/pytorch-quantization/examples/calibrate_quant_resnet50.ipynb)

The notebook "manually" imports functions from [torchvision ](https://github.com/pytorch/vision/blob/main/references/classification/train.py)(without version reference), which seem to have been updated since, thereby leading to an error when running the example.

Specifically, in the "Create data loader" section:
```python
_args = collections.namedtuple('mock_args', ['model', 'distributed', 'cache_dataset'])
dataset, dataset_test, train_sampler, test_sampler = load_data(traindir, valdir, _args(model='resnet50', distributed=False, cache_dataset=False))
```
I get the following error using the function *load_data*:
`'mock_args' object has no attribute 'val_resize_size'`

It seems that the namedtuple _args is expected to have more attributes. I checked out the *load_data* function and it expects many more attributes besides 'val_resize_size'.

Unless I imported the wrong function or forgot to run another code-block, I'd guess that the example needs to be updated to be compatible with the current torchvision functions. At the very least, the example could point to the version that was used.

This example is referenced in an Nvidia [blog](https://developer.nvidia.com/blog/achieving-fp32-accuracy-for-int8-inference-using-quantization-aware-training-with-tensorrt/). Keeping this up-to-date might help others stumbling upon this.

## Environment

**TensorRT Version**: 8.6.1

Operating System: Windows

Python Version: 3.9.12

PyTorch Version: 2.1.1+cu121

## Steps To Reproduce

Run [tools/pytorch-quantization/examples/calibrate_quant_resnet50.ipynb](https://github.com/NVIDIA/TensorRT/blob/master/tools/pytorch-quantization/examples/calibrate_quant_resnet50.ipynb)

**Comment by zerollzeng** [comment]
@ttyio ^ ^

**Comment by Zhang-Wen-chao** [comment]
> Thanks @SebastianPinzello , we will fixing this!

Your work is great! When will this bug be fixed?
By the way, will pytorch-quantization add more examples?

**Comment by Zhang-Wen-chao** [comment]
vision tag v0.9.0 is ok.

**Comment by spinzello** [comment]
It seems that things are moving quite fast in this area and the current _modus operandi_ is to create new tools instead of building on existing ones. Don't expect too much support here. For anyone looking into quantization stuff, the TensorRT Model Optimizer seems to be quite fresh. The section on PyTorch is refactored based on pytorch_quantization. You may find more going on there. Good luck!

[TensorRT Tools Overview](https://developer.nvidia.com/tensorrt-getting-started)

[Model Optimizer Documentation](https://nvidia.github.io/TensorRT-Model-Optimizer/index.html)

[Model Optimizer GitHub](https://github.com/NVIDIA/TensorRT-Model-Optimizer)
